cloud/gcp/modules/compute/main.tf

```diff
@@ -0,0 +1,69 @@
+resource "google_cloud_run_service" "civiform_application_run_service" {
+  name     = "cloudrun-civiform"
+  location = var.region
+
+  ##TODO(@ktoor): Change to correct docker image. Will need to push the image to gcr through some mechanism.
```

Comment:
question (blocking): why does the image need to be in GCR? can cloudrun not pull images from Docker Hub?

Reply:
Yes. Unfortunately, cloud run does not give us docker.io pull out of the box which is kinda sad

Reply:
aw that's a bummer